appveyor.yml

```diff
@@ -45,7 +45,7 @@ for:
 
   test_script:
   - cmd: >-
-       node node_modules/gulp/bin/gulp test-functional-local-appveyor
-
+       node node_modules/gulp/bin/gulp test-functional-local-ie
```

Comment:
Do we need to add test run for `MSEdge` browser?

```js
node node_modules/gulp/bin/gulp test-functional-local-edge
```

Reply:
There is no Edge on Appveyor, because Appveyor uses Windows Server and Microsoft considers Edge to unstable to include it in the server OS.

Reply:
Ok.